pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3936,6 +3936,12 @@
         "netcoreapp3.0": "4.0.0.0"
       }
     },
+    "System.Runtime.Intrinsics.Experimental": {
```

Comment:
You might also need to add the package entry into the description file.

Reply:
Which description file is this?

Reply:
https://github.com/dotnet/corefx/blob/master/pkg/descriptions.json

Reply:
Fixed.